src/NzbDrone.Core/ImportLists/Plex/PlexImportBase.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using NLog;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Exceptions;
+using NzbDrone.Core.Notifications.Plex.PlexTv;
+using NzbDrone.Core.Parser;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.ImportLists.Plex
+{
+    public abstract class PlexImportBase<TSettings> : HttpImportListBase<TSettings>
+    where TSettings : PlexSettingsBase<TSettings>, new()
+    {
+        public readonly IPlexTvService _plexTvService;
+        private readonly IImportListRepository _importListRepository;
+        public override ImportListType ListType => ImportListType.Plex;
+
+        protected PlexImportBase(IImportListRepository importListRepository,
+                                  IPlexTvService plexTvService,
+                                  IHttpClient httpClient,
+                                  IImportListStatusService importListStatusService,
+                                  IConfigService configService,
+                                  IParsingService parsingService,
+                                  Logger logger)
+            : base(httpClient, importListStatusService, configService, parsingService, logger)
+        {
+            _importListRepository = importListRepository;
+            _plexTvService = plexTvService;
+        }
+
+        public override ImportListFetchResult Fetch()
+        {
+            Settings.Validate().Filter("AccessToken").ThrowOnError();
+
+            var generator = GetRequestGenerator();
+            return FetchMovies(generator.GetMovies());
+        }
+
+        public override IParseImportListResponse GetParser()
+        {
+            return new PlexParser();
+        }
+
+        public override object RequestAction(string action, IDictionary<string, string> query)
```

Comment:
The function body here is a direct copy of one of the other plex handlers. It probably makes sense to shove this in a module somewhere so we have one source of truth.